app/assets/javascripts/covid19/src/webapp/components/structures-table/ViewMoreDialog.tsx

```diff
@@ -1,136 +1,84 @@
-                                ></DetailsCell>
-                            ) : (
-                                <Typography variant="caption">
-                                    {i18n.t("No {{name}} found", { name: "details" })}
-                                </Typography>
-                            )}
-                        </ModifiedAccordion>
-                    </div>
-                </Container>
-            </DialogContent>
-        </StyledDialog>
+        <Dialog open={open} onClose={onClose} title={row.title}>
+            <Container>
+                <ImgContainer>
+                    <PdbCell data={data} row={row} validationSources={data.validationSources} />
+                    <EmdbCell data={data} row={row} />
+                </ImgContainer>
+
+                <div>
+                    <ModifiedAccordion expandedAccordion={expandedAccordion} field="entities">
+                        {row.entities.length > 0 ? (
+                            <EntityCell data={data} row={row} moreDetails={false}></EntityCell>
+                        ) : (
+                            <Typography variant="caption">
+                                {i18n.t("No {{name}} found", { name: "entities" })}
+                            </Typography>
+                        )}
+                    </ModifiedAccordion>
+
+                    <ModifiedAccordion expandedAccordion={expandedAccordion} field="ligands">
+                        {row.ligands.length > 0 ? (
+                            <LigandsCell data={data} row={row} moreDetails={false}></LigandsCell>
+                        ) : (
+                            <Typography variant="caption">
+                                {i18n.t("No {{name}} found", { name: "ligands" })}
+                            </Typography>
+                        )}
+                    </ModifiedAccordion>
+
+                    <ModifiedAccordion expandedAccordion={expandedAccordion} field="organisms">
+                        {row.organisms.length > 0 ? (
+                            <OrganismCell row={row} data={data} moreDetails={false}></OrganismCell>
+                        ) : (
+                            <Typography variant="caption">
+                                {i18n.t("No {{name}} found", { name: "organisms" })}
+                            </Typography>
+                        )}
+                    </ModifiedAccordion>
+
+                    <ModifiedAccordion expandedAccordion={expandedAccordion} field="details">
+                        {row.details ? (
+                            <DetailsCell row={row} data={data} moreDetails={false}></DetailsCell>
+                        ) : (
+                            <Typography variant="caption">
+                                {i18n.t("No {{name}} found", { name: "details" })}
+                            </Typography>
+                        )}
+                    </ModifiedAccordion>
```

Comment:
4 times the same code (except for the field). Can we abstract it into a component?